docs/guides/site-to-site-connectivity/end-customers.mdx

```diff
@@ -0,0 +1,245 @@
+
+In all cases, ngrok stores information about the machine where you run your agent, such as its IP address, operating system, CPU architecture, and anonymized details about the environment.
+
+ngrok also logs changes to account configurations, such as agent start/stop, API key creation, and more. You can see the list in our [Event Store](/docs/obs/reference/) documentation.
+
+We retain data about your site-to-site connection at the direction of your vendor, who can also delete data at any time.
+
+Read our [primer on data at ngrok](https://ngrok.com/blog-post/data-at-ngrok) for more details.
+
+### What security practices does ngrok follow?
+
+ngrok uses the shared security responsibility model, where we are responsible for the security of our network and deliver features you can configure to secure your services. You and your vendor are responsible for securing your site-to-site connectivity architecture.
+
+Our fundamental security practices include access control via an identity provider, change management, full encryption at rest, and much more.
+
+See our [Security, Privacy, and Compliance](https://ngrok.com/security) and [Trust Center](https://trust.ngrok.com/) pages for details.
+
+## Operations and agent deployment
+
+### Where can I run the ngrok agent and what are the ngrok agent’s system requirements? {#where-can-i-run-the-ngrok-agent-and-what-are-the-ngrok-agent’s-system-requirements?}
+
+The ngrok agent runs on Linux, Windows, and macOS systems and most CPU architectures, which you can get from our [agent downloads page](https://ngrok.com/download).
+
+We also distribute the agent as [SDKs](/docs/agent-sdks/), a [Docker container](https://hub.docker.com/r/ngrok/ngrok), and a [Kubernetes Operator](/docs/k8s/).
+
+See our [supported platforms documentation](/docs/agent/#system-requirements) for details about supported CPU architectures and resource requirements.
+
+### How do we manage the lifecycle and maintenance of the agent?
+
+First, we recommend configuring your ngrok agent to [run in the background](/docs/agent/#background-service) as a native operating system service. This functionality works on all Linux, Windows, and macOS systems, and once installed, the ngrok service starts at boot-time, automatically restarts after crashes, and sends logs to your system’s native logging service.
+
+If you deploy the ngrok agent with Docker, you can utilize Docker’s [restart policies](https://docs.docker.com/engine/reference/run/#restart-policies-restart) or [systemd integration](https://docs.docker.com/engine/install/linux-postinstall/#configure-docker-to-start-on-boot-with-systemd); with our [Kubernetes Operator](/docs/k8s/ngrok-operator), ensure the [container restart policy](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy) is set to `Always`.
```

Comment:
If you deploy the ngrok agent with Docker, you can utilize Docker’s [restart policies](https://docs.docker.com/engine/reference/run/#restart-policies-restart) or [systemd integration](https://docs.docker.com/engine/install/linux-postinstall/#configure-docker-to-start-on-boot-with-systemd); with our [Kubernetes Operator](/docs/k8s), ensure the [container restart policy](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy) is set to `Always`.

*Updated Kubernetes Operator link